app/templates/index.html

```diff
@@ -0,0 +1,19 @@
+<html>
```

Comment:
I'm not 100% sure, but I don't think we're going to have templates or any rendering in the Golang server, so we may want to remove this.  I think all of our frontend will be served up by a Node.js server.  cc @karrde00